### PR TITLE
{lang}[GCCcore/6.4.0] Guile v2.2.2

### PR DESCRIPTION
**Pull request by JackPerdue**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by JackPerdue** [comment]
Depends on PR #4930

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 5 out of 5 (3 easyconfigs in this PR)
ca004.camp.thecrick.org - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/a9a8263ddeebdf70a76a53d984c57387 for a full test report.

File: easybuild/easyconfigs/l/libunistring/libunistring-0.9.7-GCCcore-6.4.0.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+easyblock = 'ConfigureMake'
```
Comment (verdurin) [inline easybuild/easyconfigs/l/libunistring/libunistring-0.9.7-GCCcore-6.4.0.eb]: Please remove this, now the specific `easyconfig` has been merged.

Comment (JackPerdue) [inline easybuild/easyconfigs/l/libunistring/libunistring-0.9.7-GCCcore-6.4.0.eb]: I'm not clear on what I need to change here and why.  Any additional info appreciated.

Comment (verdurin) [inline easybuild/easyconfigs/l/libunistring/libunistring-0.9.7-GCCcore-6.4.0.eb]: I meant that #4930 has been merged already, so this shouldn't be required anymore?

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
ca004.camp.thecrick.org - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/a750560bc8949996b13be5534e0f2341 for a full test report.

**Comment by verdurin** [comment]
Merging, thanks @JackPerdue

**Pull request merged**